src/main/java/nl/novi/backend_hw11_techiteasy/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,67 @@
+package nl.novi.backend_hw11_techiteasy.controllers;
+import nl.novi.backend_hw11_techiteasy.exceptions.InvalidNameException;
+import nl.novi.backend_hw11_techiteasy.exceptions.RecordNotFoundException;
+import nl.novi.backend_hw11_techiteasy.models.Television;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/televisions")
+public class TelevisionsController {
+
+    private ArrayList<Television> televisions = new ArrayList<Television>();
+
+    @GetMapping
+    public ResponseEntity<List<Television>> getAllTelevisions() {
+        if (!televisions.isEmpty()) {
+            return ResponseEntity.ok(televisions);
+        }
+        else {
+            throw new RecordNotFoundException("No televisions found");
+        }
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity <Television> getTelevisionById(@PathVariable int id) {
+        if (id >= 0 && id < televisions.size()) {
```

Comment:
Deze heeft nu geen zin meer. hij gaat de televisie opslaan in de database dus dit zou het effect weg halen...